src/app/app.component.html

```diff
@@ -0,0 +1,6 @@
+
+<app-navbar> </app-navbar>
+
+<router-outlet> </router-outlet>
```

Comment:
Good organized code here.

Reply:
Thanks! I remembered that on Saturday you mentioned the main app component should only import other components.